examples/bridge/time-series/simple.html

```diff
@@ -63,7 +63,7 @@
     const source = new carto.source.SQL(`select * from cordoba_catastro_simple where year > 1900 and year < 2018`);
     const viz = new carto.Viz(`
       @duration: 30
-      @animation: animation(linear($year, 1900, 2017), @duration, fade(0.1, ${Number.MAX_SAFE_INTEGER}))
+      filter: animation(linear($year, 1900, 2017), @duration, fade(0.1, ${Number.MAX_SAFE_INTEGER}))
```

Comment:
It's working now both in the time series and the animation controls widget. It'd actually be great to unify the methods of these widgets since they're doing mainly the same. This example works, and the other example works as well.